setup.py

```diff
@@ -42,6 +42,7 @@
     "invenio-app>=1.3.0",
     "invenio-celery>=1.2.2",
     "invenio-db>=1.0.8",
+    "invenio-pidstore>=1.2.2",
```

Comment:
question: why is this necessary?

Reply:
The `RecordItem` object the `Marc21RecordServices` returns only has the `Record` object as a private member, accessing it would be possible but bad practice. Additionally the aforementioned member somehow isn't identical with the `Record` object I would expect it to be. In order to access the proper record I would use the `Resolver` from the invenio-pidstore module, which would return the proper `Record`, so files can be attached.

Reply:
I have deleted the dependency and the reference.